docs/open-source/create-a-provider.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Create a provider
+description: A short guide on integrating Qiskit into an external provider's quantum resources.     
+---
+# Integrate external quantum resources with Qiskit
+
+The Qiskit SDK is built to support third parties in creating external providers of quantum resources. This means that any organization which develops or deploys quantum compute resources can integrate their services into Qiskit and tap into its userbase. Doing so requires creating a package which supports requests for quantum compute resources and returns them to the user. Additionally, the package must allow users to submit jobs and retrieve their results, either through an implementation of the `qiskit.primitives` or `qiskit.provider` objects.
```

Comment:
```suggestion
The Qiskit SDK is built to support third parties in creating external providers of quantum resources.
This means that any organization which deploys resources that can be used to execute or simulate the execution of quantum program, it can be integrated  into Qiskit and tap into its userbase.
Doing so requires creating a package which supports requests a description of those quantum compute resources and list them to the user.
Additionally, the package must allow users to submit jobs and retrieve their results, either through an implementation of the `qiskit.primitives` objects.
```

Reply:
Splitting this up makes sense to do thanks. I edited the page to follow this and changed the wording of the last sentence just a bit.